react-native-reanimated.d.ts

```diff
@@ -225,6 +225,10 @@ declare module 'react-native-reanimated' {
     export function concat(
       ...args: Array<Adaptable<string> | Adaptable<number>>,
     ): AnimatedNode<string>;
+    export function format(
+      value: Adaptable<number>,
+      format: Adaptable<string>
```

Comment:
The type doesn't indicate that the last argument is optional. This change should do the trick.

```diff
- format: Adaptable<string>
+ format?: Adaptable<string>
```

Reply:
Thanks for the review! And done